src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,153 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final long USER_ID = 145789L;
+    private static final String LOGIN = "Login";
+    private static final String PASSWORD = "password";
+    private static final String PASSWORD_FOR_SAME_USER = "passwordsame";
+    private static final String SHORT_PASSWORD = "pass";
+    private static final int AGE = 22;
```

Comment:
I think more informative will be create constant for `MIN_AGE`, and during testing use e.g. `MIN_AGE-1` , `MIN_AGE+1`